mappings/net/minecraft/world/chunk/ChunkManager.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_51 net/minecraft/world/chunk/ChunkManager
+	METHOD method_1801 tick ()Z
+	METHOD method_1802 hasChunk (II)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_1803 generate (Lnet/minecraft/class_51;II)V
```

Comment:
This seems to only do decoration (Features, Trees, Snow), so `decorate` would be more fitting